### PR TITLE
Fixing difference in hand ray pointing math between left and right hands

### PR DESCRIPTION
**Pull request by RogPodge**
## Overview
Hand rays had a slightly different update function based on handedness. This PR fixes the discrepancy

old:
275119702_492113852506012_5273646645051996017_n

new:
275200622_1893843437452649_4282125333137013771_n

### CODE REVIEW

**Review by keveleigh — APPROVED** [review]

File: Assets/MRTK/Tests/PlayModeTests/BoundsControlTests.cs [hunk 1]
```diff
@@ -993,11 +993,12 @@ public IEnumerator TranslateViaFarInteraction()
             yield return hand.MoveTo(endPosition);
 
             // make sure translation is as expected and no other transform values have been modified through this
-            Vector3 expectedPosition = new Vector3(1f, 0f, 1.5f);
+            Vector3 expectedPosition = new Vector3(1.1126f, 0f, 1.5f);
             Vector3 expectedSize = Vector3.one * 0.5f;
+            Debug.Log(boundsControl.transform.position.ToString("F4"));
```
Comment (keveleigh) [inline Assets/MRTK/Tests/PlayModeTests/BoundsControlTests.cs:998]: Intentionally left in?

File: Assets/MRTK/Tests/PlayModeTests/BoundsControlTests.cs [hunk 2]
```diff
@@ -993,11 +993,12 @@ public IEnumerator TranslateViaFarInteraction()
...
             TestUtilities.AssertAboutEqual(boundsControl.transform.position, expectedPosition, "cube didn't move as expected");
             TestUtilities.AssertAboutEqual(boundsControl.transform.localScale, expectedSize, "cube scaled while translating");
-            TestUtilities.AssertAboutEqual(boundsControl.transform.rotation, Quaternion.identity, "cube rotated while translating");
+            TestUtilities.AssertAboutEqual(boundsControl.transform.rotation, Quaternion.identity, "cube rotated while translating"); ;
```
Comment (keveleigh) [inline Assets/MRTK/Tests/PlayModeTests/BoundsControlTests.cs:1001]: Extra `;` at the end of this line

File: Assets/MRTK/Tests/PlayModeTests/BoundsControlTests.cs [hunk 3]
```diff
@@ -2447,16 +2448,16 @@ public IEnumerator ManipulateViaAppBarFarInteraction()
...
             Vector3 scalePoint = new Vector3(0.165f, 0.267f, 0.794f); // end position for far interaction scaling
             Vector3 appBarButtonStart = new Vector3(-0.028f, -0.263f, 0.499f); // location of hand for interaction with the app bar manipulation button after scene setup
-            Vector3 appBarButtonAfterScale = new Vector3(0.009f, -0.255f, 0.499f); // location of the hand for interaction with the app bar manipulation button after scaling
+            Vector3 appBarButtonAfterScale = new Vector3(-0.009f, -0.255f, 0.499f); // location of the hand for interaction with the app bar manipulation button after scaling
 
             // first test to interact with the cube without activating the app bar
             // this shouldn't scale the cube
             TestHand hand = new TestHand(Handedness.Left);
-            yield return hand.Show(pointOnCube); // Initially make sure that hand ray is pointed on cube surface so we won't go behind the cube with our ray
+            yield return hand.Show(pointOnCube); // Initially make sure that hand ray is pointed on cube surface so we won't go behind the cube with our ray\
```
Comment (keveleigh) [inline Assets/MRTK/Tests/PlayModeTests/BoundsControlTests.cs:2460]: Extra `\` at the end of this line

File: Assets/MRTK/Tests/PlayModeTests/BoundsControlTests.cs (hunk 2 above)
Comment (keveleigh) [reply Assets/MRTK/Tests/PlayModeTests/BoundsControlTests.cs:1001]:
```suggestion
            TestUtilities.AssertAboutEqual(boundsControl.transform.rotation, Quaternion.identity, "cube rotated while translating");
```

File: Assets/MRTK/Tests/PlayModeTests/BoundsControlTests.cs (hunk 3 above)
Comment (keveleigh) [reply Assets/MRTK/Tests/PlayModeTests/BoundsControlTests.cs:2460]:
```suggestion
            yield return hand.Show(pointOnCube); // Initially make sure that hand ray is pointed on cube surface, so we won't go behind the cube with our ray
```

**Review by keveleigh — CHANGES_REQUESTED** [review]

File: Assets/MRTK/Tests/PlayModeTests/BoundsControlTests.cs (hunk 1 above)
Comment (RogPodge) [reply Assets/MRTK/Tests/PlayModeTests/BoundsControlTests.cs:998]: Fixing these 😔 woops

**Review by keveleigh — APPROVED** [review]

**Pull request merged**